packages/office-ui-fabric-react/src/components/Modal/Modal.types.ts

```diff
@@ -118,6 +118,8 @@ export type IModalStyleProps = Required<Pick<IModalProps, 'theme'>> &
     hasBeenOpened?: boolean;
     /** Positioning of modal on first render */
     modalRectangleTop?: number;
+    /** Props for layer element */
+    layerProps?: ILayerProps;
```

Comment:
I find it surprising that CI is passing without having run `npm run update-api`. This seems to me like a type change to `IModalStyleProps` which is exported publicly 🤔

Reply:
[This](https://github.com/OfficeDev/office-ui-fabric-react/blob/7cf240e434ebf5b9961be95ec6bb2d2503b4907b/packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.ts#L13177) is concerning. It's not expanded in the API file and rather is `WARNING: Unsupported export` 😕

Reply:
@adimittal91 @natalieethell This has been a missing feature in api-extractor that Pete was going to fix for a long time. Do you know what happened with that?

Reply:
All of the "WARNING: Unsupported export" warnings should go away when we upgrade to api-extractor 7 (which is still in beta, but should be officially released soon). 

`IModalStyleProps`, specifically, looks like this in the new api.ts file generated from api-extractor 7:

```tsx
// @public (undocumented)
declare type IModalStyleProps = Required<Pick<IModalProps, 'theme'>> & Pick<IModalProps, 'className' | 'containerClassName' | 'scrollableContentClassName' | 'topOffsetFixed' | 'isModeless'> & {
    isOpen?: boolean;
    isVisible?: boolean;
    hasBeenOpened?: boolean;
    modalRectangleTop?: number;
};
```

Reply:
I should have mentioned this earlier but can you call this prop `layer` instead of `layerProps`? It will line up with the `layer` style section naming and with Slots functionality we're introduction with Fabric 7, and will help ensure this component is backwards compatible. Thanks!

Reply:
> I should have mentioned this earlier but can you call this prop `layer` instead of `layerProps`? It will line up with the `layer` style section naming and with Slots functionality we're introducting with Fabric 7, and will help ensure this component is backwards compatible. Thanks!

@aftab-hassan can you make the above change re: `layer` vs `layerProps` prior to us merging this?

Reply:
@JasonGore @KevinTCoughlin I have renamed `layerProps` to `layer`. I feel like I have misunderstood the ask. There is now a `layer` prop of type `IStyle` as well as `ILayerProps`, in the `Modal.types.ts` file, which makes it difficult to read IMO.

Reply:
Marking this comment as `Resolved`. I reverted the naming based on this comment https://github.com/OfficeDev/office-ui-fabric-react/pull/8273#discussion_r268243178